netebpfext/net_ebpf_ext_sock_ops.c

```diff
@@ -86,11 +86,56 @@ _ebpf_sock_ops_get_current_pid_tgid(
     return (sock_ops_ctx->process_id << 32 | (uint32_t)(uintptr_t)PsGetCurrentThreadId());
 }
 
+static uint64_t
```

Comment:
sockops is invoked at flow-established+deleted. right?
this must be done in the very first connect callout.

Reply:
moved to sock_addr